css-builder.js

```diff
@@ -153,14 +156,40 @@ define(['require', './normalize'], function(req, normalize) {
   }
 
   cssAPI.write = function(pluginName, moduleName, write, parse) {
+    var cssModule;
+    
     //external URLS don't get added (just like JS requires)
     if (moduleName.match(absUrlRegEx))
       return;
 
     layerBuffer.push(cssBuffer[moduleName]);
+    
+    if (!global._requirejsCssData) {
+      global._requirejsCssData = {
+        usedBy: {css: true},
+        css: ''
+      }
+    } else {
+      global._requirejsCssData.usedBy.css = true;
+    }
+
+    if (config.buildCSS != false) {
```

Comment:
Not sure why this codebase was NOT checked by jshint/eslint/jsbeautifier before, but could u please change to `!==` usage?

Note:

`jshint css-builder.js` latest from master:

```
css-builder.js: line 22, col 16, 'csso' used out of scope.
css-builder.js: line 46, col 11, 'file' is already defined.
css-builder.js: line 144, col 40, Use '===' to compare with '0'.
css-builder.js: line 152, col 4, Missing semicolon.
css-builder.js: line 158, col 4, Missing semicolon.
css-builder.js: line 173, col 8, Missing semicolon.
css-builder.js: line 178, col 25, Use '!==' to compare with 'false'.
css-builder.js: line 195, col 4, Missing semicolon.
css-builder.js: line 220, col 30, Use '!==' to compare with 'false'.
css-builder.js: line 220, col 64, Use '!==' to compare with 'true'.
css-builder.js: line 223, col 23, Use '===' to compare with ''.
css-builder.js: line 227, col 11, Misleading line break before '+'; readers may interpret this as an expression boundary.
css-builder.js: line 234, col 4, Missing semicolon.

```